rfc_backmatter.md

```diff
@@ -725,10 +725,10 @@ Residual  | Predictor w/o shift | Predictor | Sample value
 2         | -26  | -7  | -5
 0         | 1    | 0   | 0
 
-By lining all these samples up, we get the following input for the MD5 summing process.
+By lining all these samples up, we get the following input for the MD5 checksumming process.
```

Comment:
I would have said `MD5 checksum computing process`, but I am not a native English speaker.

Reply:
Well, neither am I...

It has a mention here: https://en.wiktionary.org/wiki/checksum#English but it does not seem to be frequently used. Probably better to change it indeed.

Reply:
```suggestion
By lining all these samples up, we get the following input for the MD5 checksum calculation process.
```